IOS/SwiftSeedProject/SwiftSeedProject/Interfaces/Components/MSButton.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  MSButton.swift
+//  SwiftSeedProject
+//
+//  Created by Mugu on 3/20/17.
+//  Copyright © 2017 Making Sense. All rights reserved.
+//
+
+import UIKit
+
+@IBDesignable public class MSButton: UIButton {
+
+    @IBInspectable var borderColor: UIColor = UIColor.white {
+        didSet {
+            layer.borderColor = borderColor.cgColor
+            setNeedsLayout()
+        }
+    }
+
+    @IBInspectable var borderWidth: CGFloat = 2.0 {
+        didSet {
+            layer.borderWidth = borderWidth
+            setNeedsLayout()
+        }
+    }
+    
+    @IBInspectable var rounded: Bool = true{
+        didSet {
+            if (rounded) {
+                layer.cornerRadius = radius * bounds.size.width
+            }
+            clipsToBounds = true
+            setNeedsLayout()
+        }
+    }
+    
+    @IBInspectable var radius: CGFloat = 0.5
```

Comment:
Right. I might need to make an actual method since it can be used on both properties.